modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: PROCEDURE
+
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+With {pipelines-shortname}, to run pods in the user namespace, you must pass the annotations that CRIO expects. To add these annotations for all users, you use the `oc edit clustertask buildah` command to edit the `buildah` cluster task. Or, to add the annotations to a specific namespace, you export the cluster task as a task to that namespace.
+
+Use {pipelines-shortname} to run pods in the user namespace.
+
+.Prerequisites
+
+* Pass the annotations that CRIO expects.
```

Comment:
I'm unfamiliar with this process. It seems like pas
```suggestion
* You have passed the annotations that CRIO expects.
```
Maybe add an xref to the `.Additional resources` heading (in the Assembly file, just below the include statement for this module) that links to a topic on how to pass the annotations that CRIO expects...

Reply:
Note, there still must be a list of prerequisites for procedures, so the correct prereqs should be established and added here instead